build/Targets/SettingsSdk.props

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <RoslynSdkProject>true</RoslynSdkProject>
-    <EnableDefaultItems>false</EnableDefaultItems>
+    <EnableDefaultNoneItems>false</EnableDefaultNoneItems>
```

Comment:
By default the SDK adds a None item for any file in the project directory that's not a source file or a resx. This means that any other files in the tree will start showing up in solution explorer. It's not an useful feature in the SDK and one that sometimes causes unexpected behaviors. This property simply turns off that behavior - so only Compile and EmbeddedResource items are automatically added,